src/components/title/index.js

```diff
@@ -19,10 +19,17 @@ const Title = ({ title, logo, onRequestClose }) => (
     )}
     {!logo && title && <StyledH3>{title}</StyledH3>}
 
-    <IconWrapper onClick={onRequestClose}>
-      <Close />
-    </IconWrapper>
+    {!inline && (
```

Comment:
can't we use here: 

{onRequestClose && (
  <IconWrapper onClick={onRequestClose}>
      <Close />
    </IconWrapper>
)}

Reply:
yes, we can but the inline makes more sense and it is understandable why we are doing that. So went with the `inline` prop.

Reply:
yeah, but less options is better I think for the developer.  those basically say the same thing. 
if there is a `onRequestClose` function bassed in then add a close icon, otherwise not. 
then it is completely independent of where the component is used (inline, modal or whatever)

Reply:
well, okay. In that case, we have to remove the `onRequestClose` from the `defaultProps` and add a null check where the `onRequestClose` is being called.